src/Employer/Employer.Web/Filters/LevyDeclarationCheckFilter.cs

```diff
@@ -102,6 +117,11 @@ private bool RequestIsForALevyPage(ActionExecutingContext context)
             return controllerName ==  nameof(LevyDeclarationController);
         }
 
+        private async Task<bool> HasEmployerEOI(string employerAccountId)
+        {
+            return await _recruitVacancyClient.GetEmployerEOIAsync(employerAccountId);
```

Comment:
don't need async/await, could just return the Task

Reply:
i need the return value.

Reply:
yes but you don't need async await keywords in this method as you don't need to await anything. Just `return _recruitVacancyClient.GetEmployerEOIAsync(employerAccountId);` will do and the code that calls this method can do the await.